### DISCUSSION TITLE
Possibility to deactivate Seccomp in the collabora-aio container

### DISCUSSION THREAD

**Discussion by N8schatten** [nextcloud/all-in-one#1148] [Ideas] [answered]
Since Synology Nas devices (and a few other OS possibly) do not support Seccomp, the container cannot be operated without deactivating it.
Would be nice to have a function to disable Seccom (extra_params --o:security.seccomp=false in the environment variables).

**Comment by szaimen** [comment, marked as answer]
Will be handled in https://github.com/nextcloud/all-in-one/issues/1149

**Reply by N8schatten** [reply]
Thank you very much.

**Reply by szaimen** [reply]
This is now released with v2.0.3 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Reply by N8schatten** [reply]
In the extra_params, "--o:security.seccomp=true" is now preset. But how can I set it to false permanently now?
If I change it, it will still be overwritten the next time the container is restarted.

**Reply by szaimen** [reply]
Check this out: https://github.com/nextcloud/all-in-one#how-to-disable-collaboras-seccomp-feature

**Reply by N8schatten** [reply]
Ok, works. Thank you very much
